java/year-service/build.gradle

```diff
@@ -25,10 +19,14 @@ dependencies {
 	testImplementation('org.springframework.boot:spring-boot-starter-test') {
 		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
 	}
-
-	implementation 'io.honeycomb:honeycomb-opentelemetry-java:1.0-SNAPSHOT'
 }
 
 test {
 	useJUnitPlatform()
 }
+
+task setEnvVars(type:Exec) { 
+	// expects AGENT_PATH to point to OpenTelemetry Java agent .jar file
+	// works for vanilla OpenTelemetry agent or Honeycomb OpenTelemetry agent distro
+	environment 'JAVA_TOOL_OPTIONS', '-javaagent:${AGENT_PATH}'
```

Comment:
I was torn between setting this here or in `java_year_service()` launch function in the Tiltfile. @robbkidd do you think it makes more sense to set it in the one spot like this so that all the Java services can use it?